scout/server/blueprints/variants/controllers.py

```diff
@@ -416,6 +394,43 @@ def variant_export_lines(store, case_obj, variants_query):
     return export_variants
 
 
+def variant_export_genes_info(store, gene_list):
```

Comment:
Moved this code to another function to simplify the variant_export_lines function.